src/patterns/LibraryHeader.vue

```diff
@@ -2,7 +2,7 @@
   <component :is="type" :class="['lux-library-header', theme]">
     <wrapper class="lux-header-content" :maxWidth="maxWidth">
       <library-logo :theme="theme"></library-logo>
-      <a class="lux-app-name" :href="appUrl" :title="appName">
+      <a class="lux-app-name" :href="appUrl" :title="appName" :aria-label="appName" >
```

Comment:
@hackmastera I'm updating this to use `aria-labelledby` due to the presence of visible text label on the page.